web/src/engine/osk/src/visualKeyboard.ts

```diff
@@ -832,79 +827,6 @@ export default class VisualKeyboard extends EventEmitter<EventMap> implements Ke
     return boundingRect;
   }
 
-  /**
-   * Adjusts a potential "interactive boundary" definition by enforcing an
-   * "event cancellation zone" near screen boundaries that are not directly adjacent
-   * to the ongoing input event's initial coordinate.
-   *
-   * This facilitates modeling of conventional cancellation gestures where a user would
-   * drag the mouse or touch point off the OSK, as mouse and touch event handlers receive
-   * no input beyond screen boundaries.
-   *
-   * @param baseBounds The baseline interactive bounding area to be adjusted
-   * @param startCoord The initial coordinate of a currently-ongoing input event
-   * @returns
-   */
-  private applyScreenMarginBoundsThresholding(baseBounds: BoundingRect,
```

Comment:
The inspiration behind the "safe zone padding" gesture-engine feature.  As it lives there now, we no longer need it here.